Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM registry.redhat.io/rhel8/go-toolset:1.17 AS builder
+FROM registry.redhat.io/rhel8/go-toolset:latest AS builder
```

Comment:
I think that is a good idea to stick to an specific go version. If not, in the future we can realize that, if our code is not compatible with 1.19, some day it starts failing without any reason in our end